modules/find.xq

```diff
@@ -6,7 +6,6 @@ xquery version "3.1";
  : The info parameter can be used for troubleshooting
  :)
 
-import module namespace app="http://exist-db.org/xquery/app" at "app.xqm";
```

Comment:
Because of the use of `app:redirect-to` below, I restored this module import line in a hotfix tonight, and published a new release: 2.1.1. Sorry I missed this in my final review.